src/frontend/sharedComponents/Button.tsx

```diff
@@ -54,6 +45,8 @@ const Button = ({
     >
       <TouchableNativeFeedback
         disabled={disabled}
+        // TODO: There's some typing issue with react-native-gesture-handler here. Using the definition from react native works fine
+        // @ts-expect-error
         background={TouchableNativeFeedback.Ripple(VERY_LIGHT_BLUE, false)}
```

Comment:
think there's something off about `react-native-gesture-handler`'s typedef for the `Ripple` static method. Linter gives me an error that it doesn't exist on the class but it's supposed to be there to mirror the React Native API

Reply:
upgrading to latest version of `react-native-gesture-handler` fixes this. upgrade is handled is #649 so will mark this as resolved